src/components/progress-report/dto/community-stories.dto.ts

```diff
@@ -0,0 +1,26 @@
+import { keys as keysOf } from 'ts-transformer-keys';
+import { SecuredProps } from '~/common';
+import { RegisterResource } from '~/core';
+import { PromptVariantResponse } from '../../prompts/dto';
+import { VariantOf } from '../../prompts/dto/variant.dto';
+import { ProgressReportHighlight } from './hightlights.dto';
+
+@RegisterResource()
+export class ProgressReportCommunityStory extends PromptVariantResponse<CommunityStoryVariant> {
+  static Props = keysOf<ProgressReportCommunityStory>();
+  static SecuredProps = keysOf<SecuredProps<ProgressReportCommunityStory>>();
+  static readonly Parent = import('./progress-report.entity').then(
+    (m) => m.ProgressReport
+  );
+  static Variants = ProgressReportHighlight.Variants;
```

Comment:
Do we maybe want to name `ProgressReportHighlight` a bit more generic if we're using it in `highlights` and `communityStories`?

Reply:
I'm going to leave it here for now. The variants for other sections of ProgressReports I think will be unique. I don't want to move it to a shared spot yet. Also the fact that community story's share the same variants as highlights is inconsequential.